File: java/lib/src/main/java/com/svix/api/EventTypeListOptions.java
// this file is @generated
package com.svix.api;

import com.svix.models.Ordering;

import lombok.Data;

@Data
public class EventTypeListOptions {
    /** Limit the number of returned items */
    Long limit;

    /** The iterator returned from a prior invocation */
    String iterator;

    /** The sorting order of the returned items */
    Ordering order;

    /** When `true` archived (deleted but not expunged) items are included in the response. */
    Boolean includeArchived;

    /** When `true` the full item (including the schema) is included in the response. */
    Boolean withContent;
}

File: php/src/Api/EventTypeListOptions.php
<?php

// this file is @generated
declare(strict_types=1);

namespace Svix\Api;

use Svix\Models\Ordering;

class EventTypeListOptions
{
    public function __construct(
        public readonly ?int $limit = null,
        public readonly ?string $iterator = null,
        public readonly ?Ordering $order = null,
        public readonly ?bool $includeArchived = null,
        public readonly ?bool $withContent = null,
    ) {
    }
}
